aws-sdk-core/lib/aws-sdk-core/plugins/s3_request_signer.rb

```diff
@@ -6,28 +6,15 @@ module Plugins
     # @api private
     class S3RequestSigner < Seahorse::Client::Plugin
 
-      class SigningHandler < RequestSigner::Handler
+      option :signature_version, 'v4'
```

Comment:
Stylistically, I prefer the method invocation with parenthesis, i.e. `option(:signature_version, 'v4')`